Justfile

```diff
@@ -71,12 +71,12 @@ build: build-cli
 # Run the CLI integration test suite
 @integ-tests +bats_args="": build
     flox-cli-tests --pkgdb "{{PKGDB_BIN}}" \
-     --flox "{{FLOX_BIN}}" -- {{bats_args}}
+     --flox "{{FLOX_BIN}}" {{bats_args}}
```

Comment:
This breaks `just integ-tests -f ".*multi-system.*"` for me such that I now have to pass the `--` myself. What is this fixing?

Reply:
It makes it so that when you pass a test name you run _only_ that test.
Right now you run "all tests" + "an extra named file".

If you add `--` manually in your args it will work as it used to